api/app/Http/Controllers/CspReportController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+
+class CspReportController extends Controller
+{
+    public function report(Request $request)
+    {
+        $report = $request->json()->all();
+        $message = sprintf(
```

Comment:
This works very well but I'm wondering if we're really highlighting the useful information in the fields.  I don't know that much about it but what would help us actually fix things?  violated-directive?  source-file?  We dont' have to take the first two fields - whatever number of whichever fields we want.

Reply:
Hmm, what about:

- `blocked-uri`
- `violated-directive`

:thinking:

Reply:
Alright, updated in [update highlighted report fields](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11351/commits/808b0ce47e73abab6ce62ca5577e9e73b7927e8d)